Resfeber/View Controllers/TripsViewController.swift

```diff
@@ -42,27 +44,40 @@ class TripsViewController: UIViewController {
 
         return UIBarButtonItem(customView: button)
     }()
-    
+
     weak var sideMenuDelegate: SideMenuDelegate?
 
     // MARK: - Lifecycle
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
         configureViews()
+        tripService = TripService(managedObjectContext: coreDataStack.mainContext, coreDataStack: coreDataStack)
+        NotificationCenter.default.addObserver(self, selector: #selector(loadList(notification:)), name: NSNotification.Name(rawValue: "load"), object: nil)
+
     }
 
     // MARK: - Selectors
+    @objc func loadList(notification: NSNotification) {
```

Comment:
If you aren't using the `notification`'s user info dictionary for anything, no point in having it as a parameter.